lib/tests/request-body-ref.test.ts

```diff
@@ -0,0 +1,84 @@
+import type { OpenAPIObject } from "openapi3-ts";
+import { expect, test } from "vitest";
+import { generateZodClientFromOpenAPI } from "../src";
+
+// https://github.com/astahmer/openapi-zod-client/issues/120
```

Comment:
a little typo here on the issue number, should be:

`// https://github.com/astahmer/openapi-zod-client/issues/122`

Reply:
fixed